### ISSUE TITLE
empty tags are skipped

### ISSUE THREAD

**Issue by JCarangoH** [smart-fun/XmlToJson#10]
if i have a object like this:

```
<identification>
    <nationalIdType>CC</nationalIdType>
    <nationalIdNumber>1121875556</nationalIdNumber>
    <name>Peter</name>
    <lastName>Bridges</lastName>
    <birthDate>19800101</birthDate>
    <birthPlace>Somewhere</birthPlace>
    <gender></gender>
    <issueDate>19000101</issueDate>
</identification>
```

the parser returns this:

```
{
"identification": 
    {
	"nationalIdNumber":1121875556,
	"lastName":"Bridges",
	"nationalIdType":"CC",
	"name":"Peter",
	"birthDate":19800101,
	"issueDate":19000101,
	"birthPlace":"Somewhere"
    }
}
```

the property "gender" is skipped

**Comment by smart-fun** [comment]
Hello!

Thank you for using XmlToJson and reporting this issue; and sorry for the bug.

I will investigate.

Probably I should return an empty String for empty tags, and add the possibility for the user to force the json field to another format (like to Integer for example).

Arnaud.

**Comment by smart-fun** [comment]
Hello again,

I did a fix that is available in version 1.4.0.
now empty tags are converted to empty Strings.

note that the new default behaviour for the library is to convert to String, and if you want another format then you have to specify it. If you want more info about this feature please read the release notes for 1.4.0 or the documentation.

Tell me if all is now ok for you,

thanks again for reporting the issue,

Arnaud.

**Comment by JCarangoH** [comment]
@smart-fun Hi,

I can confirm now its working properly, thanks to you for quickly response!
